packages/apps/crm/src/screens/lead/LeadFormScreen.js

```diff
@@ -215,7 +286,10 @@ const LeadFormScreen = ({navigation, route}) => {
         </View>
       </KeyboardAvoidingScrollView>
       <View style={styles.button_container}>
-        <Button title={I18n.t('Base_Save')} onPress={updateLeadAPI} />
+        <Button
+          title={I18n.t('Base_Save')}
+          onPress={idLead != null ? updateLeadAPI : crealteLeadAPI}
+        />
```

Comment:
It could also be a good idea to create a new component for this button, this way the two API call will be inside the new component and not on the screen and the screen will be more readable.